locale/de/default.cfg

```diff
@@ -24,6 +24,8 @@ confirm_deletion=Wirklich löschen
 
 import=Importieren
 export=Exportieren
+import_tasks=Import tasks
+import_blueprint=Import blueprint
```

Comment:
```suggestion
import_blueprint=Blaupause Importieren
```

Reply:
I never played Factorio in german, so I'm not sure what blueprints are referred to in the german translation.